### ISSUE TITLE
NSInvalidArgumentException: -[__NSArrayM insertObject:atIndex:]: object cannot be nil

### ISSUE THREAD

**Issue by momo-ozawa** [wordpress-mobile/WordPress-iOS#16963]
Sentry issue: 
https://sentry.io/share/issue/f7dec65da49644a999c99b962957fab3

Slack ref:
p1627881876002200-slack-C0180B5PRJ4

```
NSInvalidArgumentException

*** -[__NSArrayM insertObject:atIndex:]: object cannot be nil
```

### Setup: Disable the site stats module
1. Create a self-hosted site via Jurassic Ninja
2. On wp-admin, Click Set up Jetpack 
3. When prompted, click Approve (This establishes a full site + user connection)
4. Go to `[yoursite.com]/wp-admin/admin.php?page=jetpack_modules` and deactivate the stats module

### Setup: Disconnect Jetpack
1. On wp-admin, go to to the Jetpack > Connections > Manage Site Connections > Disconnect Jetpack
<img width="894" alt="Screen Shot 2021-08-02 at 17 36 41" src="https://user-images.githubusercontent.com/6711616/127894943-3d5a8772-1fed-404e-924e-fe2ab28395a4.png">

### Reproduce crash
1. On the app, add the self-hosted site you just created
2. Go to the Stats screen
3. The Jetpack Install screen should be displayed... Since the Jetpack connection flow is a bit flaky on the app, let's connect Jetpack through a browser (Repeat steps 2 and 3 in the **Setup: Disable the site stats module** section)
4. On the app, go back to the Stats screen and go through the Jetpack Login screen and enter your login credentials
5. 👀 App crashes once you've logged in successfully

**Comment by ovitrif** [comment]
👋 @momo-ozawa Can this issue be closed?

**Comment by momo-ozawa** [comment]
@ovitrif Yes! This was fixed by #19694 - closing this issue manually. Thanks for catching this 🙏
